www/janf04/cv05_ukol/styles.css

```diff
@@ -0,0 +1,111 @@
+* {
+    font-family: Arial, Helvetica, sans-serif
```

Comment:
chybi **box-sizing: border-box**, proto se stalo neco necekaneho u box03 (jeho sirka ma byt presne 100% stranky, ale tim ze ma jeste padding a border width, tak je o to vetsi, coz nechceme)